.github/workflows/cli-release.yaml

```diff
@@ -21,59 +21,110 @@ on:
         description: "Dry run - If checked, the release will not be created"
 
 jobs:
-  build-ubuntu20-binary:
-    name: "Build Ubuntu 20.04 binary"
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          ref: ${{ github.event.inputs.source_git_ref_override }}
-      - uses: aptos-labs/aptos-core/.github/actions/rust-setup@main
-      - name: Build CLI
-        run: scripts/cli/build_cli_release.sh "Ubuntu" "${{inputs.release_version}}"
-      - name: Upload Binary
-        uses: actions/upload-artifact@v4
-        with:
-          name: cli-builds-ubuntu-20.04
-          path: aptos-cli-*.zip
+#  build-ubuntu20-binary:
```

Comment:
Is it intended that this is all commented out?

Reply:
i didn't expect this to get in here, it should be unrelated